inference-engine/tests/functional/plugin/shared/include/single_layer_tests/range.hpp

```diff
@@ -36,4 +36,15 @@ class RangeLayerTest : public testing::WithParamInterface<RangeParams>,
     void SetUp() override;
 };
 
+class RangeNumpyLayerTest : public testing::WithParamInterface<RangeParams>,
```

Comment:
one comment, where is instantiation?

Reply:
Again, I don't see the instantiation for this test class

Reply:
Tests are not instantiated for CPU because Range op output shape would be dynamic (rank==1 but dynamic value) and it is not supported by CPU plugin. I yet implemented the class to enable other plugins, supporting dynamic shapes, to instantiate tests. 

RangeLayerTest class has instantiation, but tests are skipped, so I do not see a point to instantiate and skip.